### PR TITLE
fix: handle multiple deposits in one tx

### PR DESCRIPTION
**Pull request by tcar121293**
handle multiple deposits in one tx
## Description
Instead of mapping txHash to transferID, for each deposit we calculate the fee and store it into db

## How Has This Been Tested? Testing details.
locally

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Comment by mpetrun5** [comment]
Don't really understand how that fixes the problem. 
Also, tests are failing

**Comment by tcar121293** [comment]
> Don't really understand how that fixes the problem. Also, tests are failing

The issue was that to figure out which feeCollected event belongs to which deposit, we used txHash. In this case, the txHash is the same for all feeCollected events, so we don't know which feeCollected event belongs to which deposit event. 
Prior logic was that we mapped txHash with transferID, and when we try to insert fee, we try to insert multiple times the same transferID.
Tests are failing because we have obsolete contracts for our e2e setup. I will disable e2e tests for now(we have task to update testing environment #102)

**Comment by mpetrun5** [comment]
Updating test environment is separate with more test cases.
The E2E tests happy case should be fixed before this PR is merged.

**Review by nmlinaric — APPROVED** [review]

File: src/indexer/utils/evm/index.ts [hunk 1]
```diff
@@ -194,19 +185,32 @@ export function parseProposalExecution(
...
+export async function getFee(provider: Provider, feeHandlerRouterAddress: string, fromDomain: Domain, decodedLog: LogDescription): Promise<FeeData> {
+  try {
+    const feeRouter = gerFeeRouterContract(provider, feeHandlerRouterAddress)
+    const fee = (await feeRouter.calculateFee(
+      decodedLog.args.user as string,
+      fromDomain.id,
+      decodedLog.args.destinationDomainID as string,
+      decodedLog.args.resourceID as string,
+      decodedLog.args.data as string,
+      "0x00",
```
Comment (mpetrun5) [inline src/indexer/utils/evm/index.ts:197]: This will work when there is no fee data, but is not really a long term solution.

Comment (mpetrun5) [reply src/indexer/utils/evm/index.ts:197]: Maybe it is fine for now, but create an issue to either use Fee_Collected again or parse fee data from deposit arguments.

**Review by mpetrun5 — APPROVED** [review]

**Pull request merged**